app/src/main/res/values/strings.xml

```diff
@@ -7,7 +7,7 @@ Licensed under GNU's GPL 3, see README
 <resources>
 
     <string name="app_name">MyHackerspace</string>
-    <string name="empty"></string>
+    <string name="empty" translatable="false"></string>
```

Comment:
Why shouldn't this be translated anymore?

Reply:
There's not much value in translating an empty string 😄

IIRC this is only used because some code is parametrized using a string identifier, and you can use the "empty" string in place of an actual translation value. Otherwise, an empty string literal could be used.

Reply:
Ah lol. I thought it would bye the literal word "empty", not "" :laughing: